src/compas-editors/CompasVersions.ts

```diff
@@ -141,6 +133,25 @@ export default class CompasVersionsPlugin extends LitElement {
       });
   }
 
+  private openEditWizard(): void {
+    const wizard = wizards['SCL'].edit(this.doc.documentElement);
+    if (wizard) this.dispatchEvent(newWizardEvent(wizard));
+  }
+
+  private getCurrentVersion(): string {
+    const header = this.doc.querySelector('Header');
+    return (header && header.hasAttribute('version')
```

Comment:
`header?.hasAttribute('version')` does the same thing I believe :) No need to check if `header` is available.

This also applies to other checks in this file.

Reply:
Changed it even more to `header!.getAttribute('version') ?? 'unknown';`